src/Libraries/RevitNodes/GeometryConversion/ProtoToRevitCurve.cs

```diff
@@ -141,6 +141,13 @@ private static Autodesk.Revit.DB.Curve Convert(Autodesk.DesignScript.Geometry.Nu
                 crv.IsRational);
 
         }
+        private static Autodesk.Revit.DB.Curve CreateNurbSpline(IList<XYZ> controlPoints, IList<double> weights, IList<double> knots, int degree, bool closed, bool rational)
```

Comment:
It looks like the closed parameter is not used, maybe because it can be induced from knots, weights and degree.